packages/monitor-v2/src/bot-oo-v3/common.ts

```diff
@@ -0,0 +1,55 @@
+import { getMnemonicSigner, getRetryProvider } from "@uma/common";
+import { Signer } from "ethers";
+
+import type { Provider } from "@ethersproject/abstract-provider";
+
+export { OptimisticOracleV3Ethers } from "@uma/contracts-node";
+export { Logger } from "@uma/financial-templates-lib";
+export { getContractInstanceWithProvider } from "../utils/contracts";
+
+export interface BotModes {
+  settleAssertionsEnabled: boolean;
+}
+
+export interface BlockRange {
+  start: number;
+  end: number;
+}
+
+export interface MonitoringParams {
+  provider: Provider;
+  chainId: number;
+  runFrequency: number;
+  botModes: BotModes;
+  signer: Signer;
+}
```

Comment:
not a point to change for now but it would be a good idea in the future to refactor out common stuff from these configs into a common thing spread over all bots within the `monitor-v2`. For example I think all bots have a similar base `MonitoringParams` structure as well as some of the other setup logic.